src/main/java/L3Z1/CinemaSeats.java

```diff
@@ -0,0 +1,20 @@
+package L3Z1;
+
+public class CinemaSeats {
+    final int seatNumber;
+    final int seatRow;
+    boolean isFree = true;
```

Comment:
1. Po co isFree = true skoro ustawiamy je zawsze w konstruktorze?
2. Pola zawsze ukrywaj https://1024kb.pl/nauka-java/hermetyzacja-java/ (private) chyba, że masz konkretny powód (raczej rzadko się to zdarza), aby tego nie robić